modules/infinite-scroll/infinity.php

```diff
@@ -1243,7 +1243,9 @@ function query() {
 		// Add query filter that checks for posts below the date
 		add_filter( 'posts_where', array( $this, 'query_time_filter' ), 10, 2 );
 
-		$GLOBALS['wp_the_query'] = $GLOBALS['wp_query'] = new WP_Query( $query_args );
+		$GLOBALS['wp_the_query'] = $GLOBALS['wp_query'] = $infinite_scroll_query = new WP_Query();
+
+		$infinite_scroll_query->query( $query_args );
```

Comment:
According to the PRs description, there's a pretty big change implied in this simple diff here. Do you think we should add a comment so that no one would have a desire to roll the thing back and avoid using an additional variable? :)

Or maybe we can even get rid of the variable, and just call the `query` method on one of the globals after they are set?